deepchecks/tabular/checks/data_integrity/mixed_nulls.py

```diff
@@ -84,13 +86,18 @@ def run_logic(self, context: Context, dataset_kind) -> CheckResult:
         result_dict = {}
 
         for column_name in list(df.columns):
-            column_data = df[column_name]
-
-            string_null_counts = {value: count for value, count in column_data.value_counts(dropna=True).iteritems()
-                                  if string_baseform(value) in null_string_list}
+            column_data = (
```

Comment:
Add here a comment on why we cast categorical to object

Reply:
added